docs/guides/flux.md

```diff
@@ -110,35 +110,69 @@ So we now have an action called `userValueUpdated`, which when called dispatches
 ### Connecting a React Component to our Flux Store
 
 The store is now updating its data - but we have no React components connected to the store! Lets set one up:
+Here is an example of how to set up a connected Component. It shows several ways that a component can connect to Stores or simply be overridden with props.
```

Comment:
A minor error here or vaguely stated. The incoming key-value pair from the `props` object passed to the HOC are overridden if the `mapStateToProps` function in your example have a key-value pair of the same key. So in that way, the props have the lowest precedence.